src/cli/cli.go

```diff
@@ -165,6 +167,14 @@ func Handle() {
 		_ = log.Sync() // flush all logs in the buffer
 	}()
 
+	// ----------------------------------------------------------------------------------------
+
+	// print to stderr for poc.  should use the logger.
+	ctx, flushMetrics := events.NewMetrics(ctx, logger.Writer{Ctx: ctx})
+	defer flushMetrics()
```

Comment:
As a follow-up - should move this into operations so SDK users can also get this output